extensions/sar/README.md

```diff
@@ -56,32 +68,15 @@ properties. The table below shows the common name based on the wavelength and fr
 
 In SAR, you usually have frame start and end time. To describe this information it is recommended to use the [Datetime Range Extension Specification](../datetime-range/README.md). The center time of the frame should be specified with the `datetime` property for [STAC Items](../../item-spec/item-spec.md).
 
-### Band Object
-
-The bands contained in SAR image are dependent on the `sar:type`. For example, single look complex (SLC) data contain both phase and amplitude information of the signal. This may be provided for instance in form of complex number components (i and q bands) for each available polarization. Multilooked data (for example GRD in case of Sentinel-1) contain only amplitude and intensity bands for each polarization. Geocoded data contain radiometrically calibrated and terrain corrected data such as sigma0 or flattening gamma and may also contain angular information such as projected local incidence angle. Details about each band and the respective processing applied is given in its description.
-
-| Field Name          | Type         | Description |
-| ------------------- | ------------ | ----------- |
-| name                | string       | The name of the band. |
-| description         | string       | Description to fully explain the band, should include processing information. [CommonMark 0.29](http://commonmark.org/) syntax MAY be used for rich text representation. |
-| data_type           | string       | Specifies the type of the data contained in the band, for example `amplitude`, `intensity`, `phase`, `angle`, `sigma0`, `gamma0`. |
-| unit                | string       | The unit of measurement for the data, preferably the symbols from [SI](https://physics.nist.gov/cuu/Units/units.html) or [UDUNITS](https://ncics.org/portfolio/other-resources/udunits2/). |
-| polarization        | string\|null | The polarization of the band, either `HH`, `VV`, `HV`, `VH` or `null` if not applicable. |
-
-## Associating assets with bands
-
-Asset definitions that contain band data should reference the band index. Each asset should provide a `sar:bands` property that is an array of 0 based indexes to the correct [Band Objects](#band-object).
-
 ### Item [`Asset Object`](../../item-spec/item-spec.md#asset-object) fields
 | Field Name | Type     | Description                                  |
 | ---------- | -------- | -------------------------------------------- |
-| sar:bands  | [number] | Lists the band names available in the asset. |
+| sar:polarizations  | [string] | Lists the polarizations available in the asset, in order. Any of `HH`, `VV`, `HV`, `VH` |
```

Comment:
Do we validate this in the schema? Could be useful... 
Probably also applies to EO?!